defaults.ts

```diff
@@ -11,4 +11,7 @@ export const argDefaults = {
   dryRun: false,
   gitProvider: '',
   logLevel: 'ERROR',
+  // Below this pertains to historic Changelog builds
+  historic: false,
+  retag: true,
```

Comment:
Not Needed. Not sure i want to add this functionality in the end either as we can never be guaranteed that the tags are used the way that the system uses them.